libs/ui/inputs/src/lib/autocomplete/autocomplete.component.ts

```diff
@@ -61,6 +61,7 @@ export class AutocompleteComponent
   @Input() action: (value: string) => Observable<AutocompleteItem[]>
   @Input() value?: AutocompleteItem
   @Input() clearOnSelection = false
+  @Input() completeOnSelection = false
```

Comment:
I think this new input makes sense; but it looks like the existing components (e.g. fuzzy search) which relied on "clearOnSelection" haven't been changed to use this new input instead.

Also would it be possible to add a bit of doc in a comment to clarify what both of these do? thanks!

Reply:
> So I have to assume that all the components that were using the clearOnSelection in fact wanted an completeOnSelection in the first place ? How can we be sure about that ? Maybe what they wanted was clearOnSelection since that what was asked in the first place ? I don't get the logic here.

The logic is that we don't want to break existing stuff :)

> A doc ? It seems to be straight forward to me but ok sure. In which form ? A jsdoc comment above the function that use it or a single comment over each input ?

just a comment above the function, at the very least